src/garage/_dtypes.py

```diff
@@ -77,6 +78,8 @@ class EpisodeBatch(
         lengths (numpy.ndarray): An integer numpy array of shape :math:`(N,)`
             containing the length of each episode in this batch. This may be
             used to reconstruct the individual episodes.
+        episode_infos (dict): A dict of numpy arrays containing the episode-level
```

Comment:
```suggestion
        episode_infos (dict[str, np.ndarray]): A dict of numpy arrays containing the episode-level
```

Reply:
please add shape information to the documentation. These need to have shape :math:`(N, S^*)` (i.e. one for each episode with arbitrary shape of an individual sample.